kafka/signed_token_redeem_handler.go

```diff
@@ -38,27 +38,28 @@ func SignedTokenRedeemHandler(
 	msg kafka.Message,
 	producer *kafka.Writer,
 	server *cbpServer.Server,
-	log *zerolog.Logger,
+	logger *slog.Logger,
 ) error {
 	data := msg.Value
 	// Deserialize request into usable struct
 	tokenRedeemRequestSet, err := avroSchema.DeserializeRedeemRequestSet(bytes.NewReader(data))
 	if err != nil {
-		message := fmt.Sprintf("request %s: failed avro deserialization", tokenRedeemRequestSet.Request_id)
 		kafkaErrorTotal.Inc()
 		return handlePermanentRedemptionError(
 			ctx,
-			message,
+			fmt.Sprintf("failed avro deserialization"),
```

Comment:
suggestion: the just pass `"failed avro deserialization"` directly without the `fmt.Sprintf`